relay-config/src/config.rs

```diff
@@ -634,6 +669,47 @@ impl Config {
         Ok(config)
     }
 
+    /// Override configuration with values coming from other sources (e.g. env variables or
+    /// command line parameters)
+    pub fn override_config(
+        &mut self,
+        overrides: OverridableConfig,
+    ) -> Result<&mut Self, ConfigError> {
+        let relay = &mut self.values.relay;
+
+        if let Some(upstream) = overrides.upstream {
+            relay.upstream = UpstreamDescriptor::from_str(upstream.as_ref()).map_err(|_| {
+                ConfigError::new(PathBuf::new(), ConfigErrorKind::InvalidOverride.into())
+            })?;
+        }
+
+        if let Some(host) = overrides.host {
+            relay.host = host.parse().map_err(|_| {
+                ConfigError::new(PathBuf::new(), ConfigErrorKind::InvalidOverride.into())
+            })?;
+        }
+
+        if let Some(port) = overrides.port {
+            relay.port = port;
+        }
+
+        let processing = &mut self.values.processing;
+        if let Some(enabled) = overrides.processing {
+            processing.enabled = enabled;
+        }
+
+        //TODO finish here
+        if let Some(redis) = overrides.redis_url {
+            //processing.redis = from_str(redis.as_str());
```

Comment:
Please just support a single node configuration via the env var. Also, please remove the `serde_yaml` dependency. Same for Kafka, we just support a single bootstrap node with no special configuration.

```rust
processing.redis = RedisConfig::Single(redis);
```

Then, the env var can be set to:

```
export RELAY_REDIS_URL="redis://127.0.0.1:4711"
```